docs/Using-Fleet/MDM-setup.md

```diff
@@ -1,177 +1,185 @@
 
-### Step 2: generate an APNs certificate from Apple Push Certificates Portal
+```
+fleetctl generate mdm-apple --email <email> --org <org> 
+```
 
+### Step 2: generate an APNs certificate
 1. Log in to or enroll in [Apple Push Certificates Portal](https://identity.apple.com).
-2. Select **Create a Certificate**
+2. Select **Create a Certificate**.
 3. Upload your CSR and input a friendly name, such as "Fleet."
-4. Download the APNs certificate
-
-> Take note of the Apple ID you use to sign into Apple Push Certificates Portal. You'll need to use the same Apple ID when renewing your APNs certificate.
+4. Download the APNs certificate.
 
-### Step 3: configure Fleet with the required files
+> **Important** Take note of the Apple ID you use to sign into Apple Push Certificates Portal. You'll need to use the same Apple ID when renewing your APNs certificate.
 
-With the four generated files, we now give them to the Fleet server. 
-
-Restart the Fleet server with the contents of the APNs certificate, APNs private key, SCEP certificate, and SCEP private key in following environment variables:
+### Step 3: configure Fleet with the generated files
+Restart the Fleet server with the contents of the APNs certificate, APNs private key, SCEP certificate, and SCEP private key in the following environment variables:
 * [FLEET_MDM_APPLE_APNS_CERT_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-apple-apns-cert-bytes)
 * [FLEET_MDM_APPLE_APNS_KEY_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-apple-apns-key-bytes)
 * [FLEET_MDM_APPLE_SCEP_CERT_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-apple-scep-cert-bytes)
 * [FLEET_MDM_APPLE_SCEP_KEY_BYTES](https://fleetdm.com/docs/deploying/configuration#mdm-apple-scep-key-bytes)
 
 > You do not need to provide the APNs CSR which was emailed to you. 
 
-Confirm that Fleet is set up by visiting the "Fleet UI" or using "fleetctl."
-
-Fleet UI:
+### Step 4: confirm that Fleet is set up correctly 
 
-1. Head to the **Settings > Integrations > Mobile device management (MDM)** page.
+Use either of the following methods to confirm that Fleet is set up. You should see information about the APNs certificate such as serial number and renewal date.
 
-2. Look at the **Apple Push Certificates Portal** section.
+#### Fleet UI
 
-`fleetctl` CLI:
+Navigate to the **Settings > Integrations > Mobile device management (MDM)** page.
 
-1. Run `fleetctl get mdm-apple`.
+#### Fleetctl CLI
 
-You should see information about the APNs certificate such as serial number and renewal date. 
+```
+fleetctl get mdm-apple
+```
 
-## Renewing APNs
+## Renewing APNs 
 
-> Apple requires that APNs certificates are renewed once every year. 
-> * Be sure to do it early. If you renew after a certificate has expired, you will have to turn MDM off and back on for all macOS hosts. 
-> * Be sure to use the same Apple ID from year-to-year. If you don't, you will have to turn MDM off and back on for all macOS hosts. 
```

Comment:
I separated this into two bite-sized pieces. I left the general info where it is under the main heading for renewing APN
<img width="740" alt="image" src="https://user-images.githubusercontent.com/78363703/235540032-cd443028-670a-4a74-8850-f262f355ab04.png">

Then put the second part about Apple ID under the subsection that talks specifically about logging in to the Apple portal.

<img width="759" alt="image" src="https://user-images.githubusercontent.com/78363703/235540384-eb58426d-d40f-4855-85d4-ed2bf50f9525.png">

Let me know if you still want me to make the change.

Reply:
Yeah I think it should be above, because you should record which apple ID you used at the moment you created it. By the time you get to the renew section and it says "you should have recorded it a year ago tsk tsk!" It would be too late! Maybe the language should be more explicit, as in it tells you to record which apple ID you used.